src/core_init_atmosphere/Registry.xml

```diff
@@ -273,6 +273,9 @@
                      description="Whether to switch sea-ice threshold from 0.5 to 0.02"
                      possible_values="true or false"/>
 
+                <nml_option name="config_tsk_seaice_threshold" type="real" default_value="100." units="K"
```

Comment:
What do you think about moving this namelist option to a different namelist group from `&preproc_stages`? Also, would it be better to keep this option hidden by default to prevent users from changing it, misunderstanding its role and thinking that 100K is far colder than any SST we'd ever encounter on Earth?

Reply:
Michael: Do you have a suggestion about moving config_tsk_seaice_threshold outside of preproc_stages? I do not think that the other namelist groups would work as well? Thanks.

Reply:
Perhaps we could create a new namelist group for options that apply specifically to real-data initialization (mostly sea-ice at present). If it's alright with you, I could push some changes to this PR branch to handle this.

Reply:
since config_tsk_seaice_threshold is a physics related option, we could call the new namelist group "physics" or "physics_init". Most of those variables would remain "hidden" as they should not be changed.

Reply:
I like the name `&physics`. We can make that namelist group hidden by default.

Reply:
The `config_tsk_seaice_threshold` namelist option is now in a new `&physics` namelist group that is hidden by default.

Reply:
OK, I can create a new namelist group for config_tsk_seaice_threshold. I like &physics too. I will start working on this right away.